addons/product/models/res_company.py

```diff
@@ -16,52 +16,13 @@ def create(self, vals_list):
                 ('currency_id', '=', new_company.currency_id.id),
                 ('company_id', '=', False)
             ], limit=1)
-            if not pricelist:
-                params = {'currency': new_company.currency_id.name}
-                pricelist = ProductPricelist.create({
-                    'name': _("Default %(currency)s pricelist") %  params,
-                    'currency_id': new_company.currency_id.id,
-                })
-            self.env['ir.property']._set_default(
-                'property_product_pricelist',
-                'res.partner',
-                pricelist,
-                new_company,
-            )
+            if pricelist:
+                self.env['ir.property']._set_default(
+                    'property_product_pricelist',
+                    'res.partner',
+                    pricelist,
+                    new_company,
+                )
+                # TODO edm ask vfe: do we still want to default to an existing pricelist without
+                #  company when creating a new one?
```

Comment:
For me this file can be fully removed, we don't want this magic anymore, if they use pricelists, they should consider those currency changes and do it themselves.